### PR TITLE
dont write bytecode

### PR DESCRIPTION
**Pull request by marscher**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,8 +13,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 1
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
```
Comment (ocefpaf) [inline recipe/meta.yaml:17]: @marscher I am just curious what is your motivation for this as the byte code is in most of the times desirable.

Comment (marscher) [reply recipe/meta.yaml:17]: Because of issues with Python-3.5 conda-forge/conda-forge.github.io#264

Comment (ocefpaf) [reply recipe/meta.yaml:17]: Thanks! Can you xref that in your next PRs. (I see you are sending a few of those now.)

**Comment by marscher** [comment]
I can ref it in the comment section.

**Pull request merged**